admin/package.json

```diff
@@ -3,8 +3,8 @@
   "private": true,
   "version": "0.1.0",
   "engines": {
-    "node": ">=16.12.0",
-    "npm": ">=8.3.1"
+    "node": ">=20.11.0",
+    "npm": ">=10.2.4"
   },
```

Comment:
Do we get any value from having the `engines` configured in these sub-package.json files?  Kind of annoying to have to make the same change to all of them when a dev will still get warned when running against the top-level....

Reply:
You're absolutely right. I removed them from the web applications (admin and webapp). I think there's still value in having them for the node services, because if anyone (not just a dev) tries to run api on Node 16 this will give them a warning. It shouldn't be possible if it's being run in our official docker image but you never know...